google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseClient.java

```diff
@@ -36,6 +36,10 @@ default Dialect getDialect() {
     throw new UnsupportedOperationException("method should be overwritten");
   }
 
+  default String getDatabaseRole() {
```

Comment:
```suggestion
  /**
   * Returns the {@link DatabaseRole} used by the client connection. The database role that is used
   * determines the access permissions that a connection has. This can for example be used to create
   * connections that are only permitted to access certain tables.
   */
  default String getDatabaseRole() {
```